RELEASE_PROCESS.md

```diff
@@ -39,9 +39,19 @@ dnf makecache
 dnf upgrade tektoncd-cli
 ```
 
-- Make an update for [homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/tektoncd-cli.rb) for tektoncd-cli formula, and make sure you get the proper `sha256sum` and update it like this:
-
-  https://github.com/Homebrew/homebrew-core/pull/46492
+- Make an update for [homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/tektoncd-cli.rb) for tektoncd-cli formula.
+
+  * Make sure you get the proper `sha256sum` You need to download the `Source Code (tar.gz)` from the release and hit below command for the respective release
```

Comment:
```suggestion
  * Make sure you get the proper `sha256sum`. You need to download the `Darwin_x86_64.tar.gz` from the [release section of the cli GitHub](https://github.com/tektoncd/cli/releases) and run the command below for the release version you are working on:
```

Reply:
But I have addressed the other changes, but it is Source Code tarball not Darwin

Reply:
I guess it can be either, so works for me.

Reply:
No, Darwin has different `sha` and CI failed for that